semaphorejs/contracts/Semaphore.sol

```diff
@@ -136,4 +134,8 @@ contract Semaphore is Verifier, MultipleMerkleTree, Ownable {
     function setExternalNullifier(uint256 new_external_nullifier) public onlyOwner {
       external_nullifier = new_external_nullifier;
     }
+
+    function disablePermissioning() public onlyOwner {
```

Comment:
Should we have `setPermissioning(bool _newPermission)` instead? In case there is a need to re-enable permissioning after it's disabled?